opensrp-chw-core/src/main/java/org/smartregister/chw/core/dao/FHIRBundleDao.java

```diff
@@ -28,14 +30,21 @@ protected FHIRBundleModel fetchFHIRDateModel(Context context, String childBaseEn
             model.setDob(userProfile.getMiddle());
             model.setAgeInDays(userProfile.getLeft());
         }
-        //Todo: these values needs to be query and set into model
-        model.setPractitionerId(null);
-        model.setPatientId(null);
-        model.setUserName(null);
-        model.setLocationId(null);
-        model.setUniqueIdGeneratedForThinkMD(null);
+        model.setUniqueIdGeneratedForThinkMD(getRandomGeneratedId());
+        model.setPatientId(model.getUniqueIdGeneratedForThinkMD());
```

Comment:
Where is `patientId` used

Reply:
patientId is used in thinkmd lib to generate the FHIR bundle. you can find this in FHIRBundleGenerator class subject reference Patient/patientId